doc/devdocs/modules/FileExplorer/monaco/readme.md

```diff
@@ -0,0 +1,64 @@
+# Developer Preview (Monaco)
+
+Developer preview is based on [Microsoft's Monaco Editor](https://microsoft.github.io/monaco-editor/) which is maintained by the Visual Studio Code team.
+
+## Update Monaco Editor
+
+1. Download Monaco editor with [npm](https://www.npmjs.com/): `npm i monaco-editor`.
+2. Delete everything except the `min` folder (the minimised code) from the downloaded files.
+3. Copy the `min` folder into the [`monacoSRC`](/src/modules/previewpane/MonacoPreviewHandler/monacoSRC) folder.
+4. Generate the JSON file as described in the [Generate monaco_languages.json file](#generate-monaco_languagesjson-file) section.
```

Comment:
<img width="224" alt="image" src="https://user-images.githubusercontent.com/57057282/227154483-850a59c7-8b39-47d5-9d98-25b66873fe66.png">

This doesn't look very nice to me :) At least move `file` out of the link